lib/router.js

```diff
@@ -219,6 +219,13 @@ Router.prototype._handleSwaggerPathSpec = function(node, pathspec,
         });
     }
 
+    var security = pathspec.security;
```

Comment:
@gwicke but we know the module's security restrictions only after we load it. And may be it loads submodules etc..